src/ui/public/angular-bootstrap/datepicker/year.html

```diff
@@ -1,15 +1,72 @@
-<table role="grid" aria-labelledby="{{uniqueId}}-title" aria-activedescendant="{{activeDateId}}">
+<table
```

Comment:
I agree, but I don't think angular-bootstrap's datepicker directive surfaces this information. We have to update the logic of this directive to support highlighting the current range. I'd also like to get rid of the silly way the datepicker highlights the days you _can't_ support. Dave's working on a new datepicker design, so I think we can address these problems when we retrofit that design.